packages/calypso-color-schemes/package.json

```diff
@@ -25,7 +25,7 @@
 	},
 	"scripts": {
 		"clean": "npx rimraf js css src/__color-studio",
-		"prepare": "node bin/prepare-sass-assets.js && node bin/build-css.js"
+		"prepare": "yarn node bin/prepare-sass-assets.js && yarn node bin/build-css.js"
```

Comment:
I did it because https://yarnpkg.com/getting-started/migration#call-your-scripts-through-yarn-node-rather-than-node, but reading it again looks like it is not necessary for scripts.